app/(admin)/admin/notifications/page.tsx

```diff
@@ -1,16 +1,71 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import { JSX } from 'react';
+'use client';
+import { Button } from '@/components/Button/Button';
+import { getCurrentLeague } from '@/api/apiFunctions';
+import { Input } from '@/components/Input/Input';
+import { JSX, useState } from 'react';
+import { Label } from '@/components/Label/Label';
+import { sendEmailNotifications } from './actions/sendEmailNotification';
+import React from 'react';
 
 /**
  * The admin home page.
  * @returns The rendered AdminHome page.
  */
 const AdminNotifications = (): JSX.Element => {
+  const [content, setContent] = useState<string>('');
+  const [groupEmailTest, setGroupEmailTest] = useState<string[]>([]);
+  const [subject, setSubject] = useState<string>('');
+
+  /**
+   * To grab all users from the league.
+   */
+  const getLeagueData = async (): Promise<void> => {
+    const leagueId = '66c6618900033d179dda';
+    const leagueData = await getCurrentLeague(leagueId);
+    setGroupEmailTest(leagueData.participants);
+    try {
+    } catch (error) {
+      throw new Error('Error Fetching Data');
```

Comment:
Maybe we should add error in this string as well, so the user can see the specifics of the error?